CMakeLists.txt

```diff
@@ -79,21 +88,29 @@ option(ENABLE_GCOV "gcov code coverage feature" OFF)
 option(ENABLE_SANITIZE "Sanitize tool" OFF)
 option(ENABLE_SECURITY "Security Ford protocol protection" ON)
 option(ENABLE_HMI_PTU_DECRYPTION "Policy table update parsed by hmi" ON)
-option(USE_COTIRE
-       "Use Cotire to speed up build (currently only for commands tests)" ON)
+option(
+    USE_COTIRE
+    "Use Cotire to speed up build (currently only for commands tests)"
+    ON
+)
 option(USE_GOLD_LD "Use gold linker intead of GNU linker" ON)
 option(USE_CCACHE "Turn on ccache usage" ON)
 option(USE_DISTCC "Turn on distributed build_usage" OFF)
 
-set(EXTENDED_POLICY "PROPRIETARY"
-    CACHE STRING "Policy mode (PROPRIETARY, HTTP or EXTERNAL_PROPRIETARY)")
-set_property(CACHE EXTENDED_POLICY
-             PROPERTY STRINGS
-                      PROPRIETARY
-                      HTTP
-                      EXTERNAL_PROPRIETARY)
+set(
```

Comment:
@MaximRybak  in one line `set(EXTENDED_POLICY "PROPRIETARY"`

Reply:
If the line contains a keyword and the arguments are not placed in one line, then it makes the transfer of all the arguments to a new line. In the configuration file, you can set the property --max-subargs-per-line

Reply:
@MaximRybak  please make this default

Reply:
@MaximRybak  please make this default